src/main/java/com/royalrangers/controller/PlatoonController.java

```diff
@@ -0,0 +1,100 @@
+package com.royalrangers.controller;
+
+import com.dropbox.core.DbxException;
+import com.royalrangers.dto.PlatoonDto;
+import com.royalrangers.dto.ResponseResult;
+import com.royalrangers.exception.PlatoonRepositoryException;
+import com.royalrangers.service.DropboxService;
+import com.royalrangers.service.PlatoonService;
+import com.royalrangers.utils.ResponseBuilder;
+import io.swagger.annotations.ApiOperation;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+
+@Slf4j
+@RestController
+@RequestMapping("/platoon")
+public class PlatoonController {
+    @Autowired
+    private DropboxService dropboxService;
+    @Autowired
+    private PlatoonService platoonService;
+
+    @GetMapping("/get")
```

Comment:
We don't need this annotation at all. 
If do GET "/platoon" I should get list of all platoons.